app/src/androidTest/java/org/mozilla/fenix/syncintegration/SyncIntegrationTest.kt

```diff
@@ -82,6 +84,40 @@ class SyncIntegrationTest {
         }
     }
 
+    @Test
+    fun checkLoginsFromDesktopTest() {
+        homeScreen {
+        }.openThreeDotMenu {
+        }.openSettings {
+            // Necessary to scroll a little bit for all screen sizes
+            TestHelper.scrollToElementByText("Logins and passwords")
```

Comment:
Maybe move this scroll portion into the `openLoginsAndPasswordSubMenu` method, since any time you need to open that subMenu, you will most likely have to scroll.

Reply:
Makes sense, I added it to Settings Robot though since it is there where we need to scroll so that the Logins And Passwords menu is accessible